core/modules/views/tests/views_test_entity/views_test_entity.entity.inc

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * @file
+ * Test entity classes.
+ */
+
+/**
+ * Test class for the test_entity entity type.
+ */
+class BulkTestEntity extends Entity {
```

Comment:
`class BulkTestEntity extends Entity` ... strange, that's the same class as above. :thinking:

Reply:
Oops! Yes, that's wrong. I forgot to remove the class from views_test.entity.inc 😓 

Thanks for catching!

Reply:
I have removed the file views_test.entity.inc as it not needed. The class definition was moved to views_test_entity.entity.inc (and a new helper module)